lib/guest_installation_and_configuration_base.pm

```diff
@@ -2572,12 +2572,14 @@ sub start_guest_installation {
     my $_guest_installation_log = "$common_log_folder/$self->{guest_name}/$self->{guest_name}" . "_installation_log_" . $_start_installation_timestamp;
     assert_script_run("touch $_guest_installation_log && chmod 777 $_guest_installation_log");
     # Dry run always timeout when downloading initrd from download.opensuse.org in O3
-    my $ret = script_run("set -o pipefail; $self->{virt_install_command_line_dryrun} 2>&1 | tee -a $_guest_installation_dryrun_log", timeout => 600 / get_var('TIMEOUT_SCALE', 1), die_on_timeout => 0);
+    my $ret = script_run("set -o pipefail; timeout 580 $self->{virt_install_command_line_dryrun} 2>&1 | tee -a $_guest_installation_dryrun_log", timeout => 600);
     save_screenshot;
-    unless (defined(script_run('set +o pipefail', die_on_timeout => 0))) {
+    enter_cmd "set +o pipefail; echo DONE > /dev/$serialdev";
+    unless (defined(wait_serial('DONE', timeout => 30))) {
         reconnect_when_ssh_console_broken;
         script_run("set +o pipefail");
     }
+    script_run 'set +o pipefail';
```

Comment:
Why one more
```
script_run 'set +o pipefail';
```
outside unless is needed ?

Reply:
That should not be there. Fixed.